components/playlist/browser/resources/media_detector/index.js

```diff
@@ -115,9 +113,9 @@
   }
 
   function getMediaDurationInSeconds (node) {
-    let duration = node.duration
-    const isNan = (value) => { return typeof value === 'number' && Number.isNaN(value) }
-    const isInfinite = (value) => { return typeof value === 'number' && (value === Infinity || value === -Infinity) }
+    const duration = node.duration
+    const isNan = (value) => typeof value === 'number' && Number.isNaN(value)
+    const isInfinite = (value) => typeof value === 'number' && (value === Infinity || value === -Infinity)
```

Comment:
Continued from : https://github.com/brave/playlist-component/pull/10#discussion_r1072951112

This part is from https://github.com/brave/brave-ios/blob/bb1832c245c0e22d1876a544e2cde4a2356863ef/Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/PlaylistScript.js#L9

Hi, @Brandon-T . I was wondering why you're not using Number.isNan() and Number.isInfinite() instead of these functions. Is there any edge cases?